src/components/Picker.js

```diff
@@ -152,7 +152,7 @@ function Picker({
     dropDownDirection = DROPDOWN_DIRECTION.DEFAULT,
     disableLocalSearch = false,
     theme = THEMES.DEFAULT,
-    testID = {testID}
+    testID
```

Comment:
cleaner, nice

Reply:
Does it work without the change? i'm confused 😶

Reply:
untested but I *assumed* it would (dangerous of course), I'm used to typescript and a CI `tsc` run to validate things and save my silly syntax errors from ever seeing the light of day

Reply:
I did a simple test:
```js
function test({name = {name}}) {
  console.log(name);
}
undefined
test()
VM359:1 Uncaught TypeError: Cannot read properties of undefined (reading 'name')
    at test (<anonymous>:1:16)
    at <anonymous>:1:1
```

Reply:
I forgot this one:
```js
test({})
VM84:1 Uncaught ReferenceError: Cannot access 'name' before initialization
    at test (<anonymous>:1:24)
    at <anonymous>:1:1
```